pkg/collector/corechecks/oracle-dbm/activity.go

```diff
@@ -455,75 +453,54 @@ func (c *Check) SampleSession() error {
 		obfuscate := true
 		var hasRealSQLText bool
 		if sample.Statement.Valid && sample.Statement.String != "" && !previousSQL {
-
 			// If we captured the statement, we are assigning the value
 			statement = sample.Statement.String
+			hasRealSQLText = true
+		} else if previousSQL && sample.PrevSQLFullText.Valid && sample.PrevSQLFullText.String != "" {
+			statement = sample.PrevSQLFullText.String
+			hasRealSQLText = true
+		} else if (sample.OpFlags & 8) == 8 {
+			statement = "LOG ON/LOG OFF"
+			obfuscate = false
+		} else if commandName != "" {
+			statement = commandName
+		} else if sessionType == "BACKGROUND" {
+			// The program name can contain an IP address, so we have to obfuscate it
+			statement = program
+		} else if sample.Module.Valid && sample.Module.String == "DBMS_SCHEDULER" {
+			statement = sample.Module.String
+			obfuscate = false
+		} else {
+			log.Warnf("activity sql text empty for %#v \n", sample)
```

Comment:
Does this have the potential to flood our logs? How often to you expect this case to happen?

Reply:
> As I understand, INFO log messages are stopped being logged after the first N check runs, so we wouldn't capture the information afterwards.

ah I didn't know that, what about setting this as a `DEBUG`? It just feels a bit odd to call this a warning, because its not really affecting the overall check behavior, and from my understanding there isn't anything the customer can do to prevent this. Seems like its more helpful for us to debug why sql text is missing in our UIs?

Reply:
Change to `DEBUG` now.